test/redux/alert_test.js

```diff
@@ -59,6 +59,56 @@ describe("alert", () => {
       })
     })
 
+    describe("when the action is SHOW_STAGE_HELP", () => {
+      const initialState = { headerText: "Warning!", bodyText: "You're being watched." }
+      const stageConfigs = {
+        daybreak: {
+          help: {
+            headerText: "I need help!",
+            bodyText: "Tell me what to do",
+          },
+        },
+        nightfall: {
+          no_help: {},
+        },
+        stageLackingHelpConfig: {},
+      }
+
+      deepFreeze(initialState)
+      deepFreeze(stageConfigs)
+
+      describe("when the given stage has a help in the given configuration map", () => {
+        const action = {
+          type: "SHOW_STAGE_HELP",
+          retro: {
+            stage: "daybreak",
+          },
+          stageConfigs,
+        }
+
+        it("returns the help for the given stage", () => {
+          expect(reducer(initialState, action)).to.deep.equal({
+            headerText: "I need help!",
+            bodyText: "Tell me what to do",
+          })
+        })
+      })
+
+      describe("when the given stage does NOT have a help in the given configuration map", () => {
+        const action = {
+          type: "SHOW_STAGE_HELP",
+          retro: {
+            stage: "nightfall",
+          },
+          stageConfigs,
+        }
+
+        it("returns the help for the given stage", () => {
```

Comment:
we'll need to update the test language here to reflect that we expect null

Reply:
👍